ansible_collections/arista/avd/molecule/eos_validate_state/intended/test_catalogs/dc1-wan1-catalog.yml

```diff
@@ -178,22 +178,64 @@ anta.tests.ptp:
 - VerifyPtpOffset: null
 - VerifyPtpPortModeStatus: null
 anta.tests.routing.bgp:
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: evpn
+      peers:
+      - 10.255.255.0
+    result_overwrite:
+      custom_field: 'BGP EVPN Peer: dc1-leaf1a (IP: 10.255.255.0)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: evpn
+      peers:
+      - 10.255.255.2
+    result_overwrite:
+      custom_field: 'BGP EVPN Peer: dc1-leaf1b (IP: 10.255.255.2)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: path-selection
+      peers:
+      - 10.255.255.0
+    result_overwrite:
+      custom_field: 'BGP PATH-SELECTION Peer: dc1-leaf1a (IP: 10.255.255.0)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: path-selection
+      peers:
+      - 10.255.255.2
+    result_overwrite:
+      custom_field: 'BGP PATH-SELECTION Peer: dc1-leaf1b (IP: 10.255.255.2)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: link-state
+      peers:
+      - 10.255.255.0
+    result_overwrite:
+      custom_field: 'BGP LINK-STATE Peer: dc1-leaf1a (IP: 10.255.255.0)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: link-state
+      peers:
+      - 10.255.255.2
+    result_overwrite:
+      custom_field: 'BGP LINK-STATE Peer: dc1-leaf1b (IP: 10.255.255.2)'
 - VerifyBGPSpecificPeers:
     address_families:
     - afi: ipv4
       peers:
       - 10.255.255.0
-      safi: unicast
+      safi: sr-te
     result_overwrite:
-      custom_field: 'BGP IPv4 Unicast Peer: dc1-leaf1a (IP: 10.255.255.0)'
+      custom_field: 'BGP IPv4 Sr-te Peer: dc1-leaf1a (IP: 10.255.255.0)'
```

Comment:
Same as others, `BGP IPv4 SR-TE` would be cleaner :)

Reply:
Also, is this expected that this peer (10.255.255.0) is now `sr-te` instead of `unicast`?